faqs.html

```diff
@@ -5,29 +5,42 @@
 <div class="container white-text">
   <h1 class="text-accent-4 center">Frequently Asked Questions</h1>
   <ul class="collapsible black">
+    <li>
+      <div class="collapsible-header black clemson-orange-text">COVID-19?</div>
+      <div class="collapsible-body">
+      <span class="card-title clem"><i class="material-icons">error_outline</i></span>
+        <p> As passionate as we are about cybersecurity, safety is our number one priority. We sincerely hoped that the situation around COVID-19 would improve enough to continue our event on Clemson University’s campus. However, based on local health experts and community leader recommendations, <strong>it has been determined that CUCTF 2020 will be a completely virtual event on October 3rd</strong>. We refuse to be set back from this announcement and will use this opportunity to:</p>
```

Comment:
We normalized wording on the website to "cyber security" as two words a bit back so let's split it up here.

```suggestion
        <p> As passionate as we are about cyber security, safety is our number one priority. We sincerely hoped that the situation around COVID-19 would improve enough to continue our event on Clemson University’s campus. However, based on local health experts and community leader recommendations, <strong>it has been determined that CUCTF 2020 will be a completely virtual event on October 3rd</strong>. We refuse to be set back from this announcement and will use this opportunity to:</p>
```